engine/flink/executor/src/main/scala/pl/touk/nussknacker/engine/process/registrar/ForceSyncInterpretationDeterminer.scala

```diff
@@ -0,0 +1,23 @@
+package pl.touk.nussknacker.engine.process.registrar
+
+import pl.touk.nussknacker.engine.flink.api.ConfigGlobalParameters
+import pl.touk.nussknacker.engine.graph.node.{Enricher, NodeData, Processor}
+import pl.touk.nussknacker.engine.splittedgraph.{SplittedNodesCollector, splittednode}
+
+private[registrar] case class ForceSyncInterpretationDeterminer(configParameters: Option[ConfigGlobalParameters]) {
+
+  def forNode(splittedNode: splittednode.SplittedNode[NodeData]): Boolean = {
+    val forceSyncInterpretationEnabled = configParameters.flatMap(_.forceSyncInterpretationForSyncScenarioPart).getOrElse(false)
```

Comment:
Can it be turned on by default? It looks like the main reason why this flag exits is to make possible rollback to old behavior in case of some exceptional situation. Isn't it? If so, making it opt-out instead of opt-in will cause that it will be wider tested on other environments and this main reason still will be fulfilled.